### PR TITLE
Feat/docker

### LINKED ISSUE #11: ### setup docker (8sp)
- make dockerfile
- create action to push on docker hub

### PR DESCRIPTION
**Pull request by aspirio187**
## Description

Fixed few bugs when launching the app from scratch. I updated the documentation with a complete `How to install`.

Fixes #11 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

As always, the app has been manually tested. But this time it has also been tested within a Docker container with the rest of the services.

- [x] Stand alone
- [x] Docker container

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain: Docker
* SDK: .NET 6

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/docker-image.yml
Comment (nbittich) [inline .github/workflows/docker-image.yml]: should be remove before merging

File: DropboxSync.UIL/DropboxSync.UIL.csproj
Comment (nbittich) [inline DropboxSync.UIL/DropboxSync.UIL.csproj]: auto generation of dockerfile no longer needed so i guess this can be removed?

File: DropboxSync.UIL/DropboxSync.UIL.csproj
Comment (nbittich) [inline DropboxSync.UIL/DropboxSync.UIL.csproj]: auto generation of dockerfile no longer needed so i guess this can be removed?

File: DropboxSync.UIL/Properties/launchSettings.json
Comment (nbittich) [inline DropboxSync.UIL/Properties/launchSettings.json]: auto generation of dockerfile no longer needed so i guess this can be removed?

File: README.md
Comment (nbittich) [inline README.md]: should be instead:
```
image: nbittich/dropbox-sync
```

**Review by nbittich — COMMENTED** [review]
Looks very good! Thanks for the great work. Added some small comments, but overall seems fine & we tested it together.

Can be merged after comments are implemented / discussed.

File: .github/workflows/docker-image.yml
Comment (nbittich) [reply .github/workflows/docker-image.yml]: Branch 'feat/docker' can be removed 👍

File: DropboxSync.UIL/DropboxSync.UIL.csproj
Comment (nbittich) [reply DropboxSync.UIL/DropboxSync.UIL.csproj]: This one can also be removed right?

Comment (aspirio187) [reply DropboxSync.UIL/DropboxSync.UIL.csproj]: Yep it can, I will remove all useless Docker references from the project

**Review by nbittich — APPROVED** [review]

**Pull request merged**
